docs/tutorials/demo-try.md

```diff
@@ -413,7 +413,7 @@ http://deviceshifu-socket.deviceshifu.svc.cluster.local/cmd; echo
   Q: What is a MQTT? <br/>
   A: MQTT is a messaging protocol based on the publish/subscribe paradigm under the ISO standard, please click <a href="https://en.wikipedia.org/wiki/MQTT">here</a> for details. <br/>
   Q: How to interact with the MQTT in this demo? <br/>
-  A: When MQTT's digital twin receives the mqtt_data command, it returns the last message in the subscribed channel.
+  A: We support multi-channel subscription, so multiple commands can be sent to MQTT's digital twin (eg: get_topicmsg1, get_topicmsg2), When MQTT's digital twin receives the mqtt_data command, it returns the last message in the subscribed channel.
```

Comment:
multi-channel => multi-topic
so multiple commands can be sent to MQTT's digital twin => so we can have multiple APIs for MQTT's digital twin